src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,37 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        System.out.println("Creating first book:");
+        System.out.println(bookDao.create(new Book("Book1", BigDecimal.valueOf(1.99))));
+
+        System.out.println("\nFinding first book by id:");
+        System.out.println(bookDao.findById(1L));
```

Comment:
```suggestion
        Book book = bookDao.create(new Book("Book1", BigDecimal.valueOf(1.99)));
        System.out.println(book);

        System.out.println("\nFinding first book by id:");
        System.out.println(bookDao.findById(book.getId()));
```
Id is autoincrement value so better get this variable from the saved book. Otherwise, you may get an exception